package-lock.json

```diff
@@ -35,6 +35,211 @@
         "node": ">=4.4.0"
       }
     },
+    "node_modules/@babel/cli": {
```

Comment:
I don't think we should be seeing changes in `package-lock.json` if there are no changes in `package.json`, does this need a revert maybe?

Reply:
This is because the `package-lock.json` file is out of date. I haven't changed any packages. All I did was clone the repo and run `npm install`. There is a risk of human error because there's no automated workflow on this repo to check the lock file. I'm not sure what the normal procedure is. I could revert the file, but then the next person to make a change on the repository will have the same issue.

Reply:
It's probably because your version of npm is different from the npm used for to generate the file. Likely because the `nvmrc` points at `lts/*` rather than a specific version. Worth updating deps, but ideally in a separate PR.